src/main/java/org/opentripplanner/middleware/utils/FileUtils.java

```diff
@@ -31,4 +42,64 @@ public static String getFileContents(String pathAndFileName) throws IOException
         FileInputStream fileInputStream = new FileInputStream(pathAndFileName);
         return IOUtils.toString(fileInputStream, StandardCharsets.UTF_8);
     }
+
+    /**
+     * Writes the file contents to a file and then writes that file to a zip file.
+     * For info: https://stackoverflow.com/questions/14462371/preferred-way-to-use-java-zipoutputstream-and-bufferedoutputstream/17190212#17190212
+     */
+    public static void writeFileToZip(String zipPathAndFileName, String fileName, String contents) throws IOException {
+
+        ZipOutputStream zos = null;
+        try {
+            zos = new ZipOutputStream(new BufferedOutputStream(new FileOutputStream(zipPathAndFileName)));
+            zos.putNextEntry(new ZipEntry(fileName));
+            zos.write(contents.getBytes());
+            zos.closeEntry();
+        } finally {
+            try {
+                if (zos != null) zos.close();
+            } catch (IOException e) {
+                LOG.error("Cannot close zip output stream.");
+            }
+        }
+    }
+
+    /**
+     * Delete file on disk.
+     */
+    public static void deleteFile(File pathAndFileName) throws IOException {
+        Files.deleteIfExists(pathAndFileName.toPath());
+    }
+
+    /**
+     * Gets the operating system's temp directory.
+     */
+    public static File getTempDirectory() {
+        return new File(System.getProperty("java.io.tmpdir"));
+    }
+
+    /**
+     * Extracts the contents of a file contented within a zip file.
+     */
+    public static String getContentsOfFileInZip(String zipFileNameAndPath, String fileName) throws IOException {
```

Comment:
It appears this is only used during testing which is good since it attempts to read the entire file into memory which could result in an out-of-memory error if used on a massive zip file. I'd add a warning to the JavaDoc of the class noting this just in case.